test/python/WMCore_t/WMSpec_t/Steps_t/Executors_t/StageOut_t.py

```diff
@@ -46,7 +46,7 @@ def setUp(self):
         self.testDir = self.testInit.generateWorkDir()
 
         # shut up SiteLocalConfig
-        os.environ['CMS_PATH'] = os.getcwd()
+        os.environ['SITECONFIG_PATH'] = os.getcwd()
```

Comment:
Same comment as above with [StageOutMgr_t.py]

Reply:
Sorry, Kenyi. I missed this one. Fixing now.